x-pack/plugins/ml/public/components/rule_editor/utils.js

```diff
@@ -55,8 +56,9 @@ export function isValidRule(rule) {
       isValid = true;
     } else {
       const scope = rule.scope;
-      if (scope !== undefined && Object.keys(scope).length > 0) {
-        isValid = true;
+      if (scope !== undefined) {
+        const enabledScope = Object.keys(scope).find(field => (scope[field].enabled === true));
```

Comment:
This could use `.some(...)` instead to be able to just evaluate against a boolean.